msg/EntityFilters.msg

```diff
@@ -0,0 +1,20 @@
+# A set of filters to apply to entity queries. See GetEntities, GetEntitiesStates.
+
+string filter                                       # Optional, defaults to empty. Return entities with matching names.
+                                                    # Entity names are matched with the filter regular expression.
+                                                    # An empty filter will result in all entities being returned.
+                                                    # The regular expression syntax is POSIX Extended,
+                                                    # see https://pubs.opengroup.org/onlinepubs/9799919799/ definitions.
+                                                    # Applies together with other filters (categories, tags).
+uint8[] categories                                  # Optional, defaults to empty, which means no category filter.
+                                                    # Entities matching any of the categories will be returned.
+                                                    # To get entity category, use GetEntityInfo.
+                                                    # Applies together with other filters (filter, tags).
+TagsFilter tags                                     # Tags filter to apply. To get entity tags, use GetEntityInfo.
```

Comment:
Tags for entities are part of spawnables. There is no way to apply tags to entity during /after spawning. The same is for categories. Please correct me if I am wrong @adamdbrw .